ports/mimxrt10xx/common-hal/busio/UART.c

```diff
@@ -229,16 +285,16 @@ void common_hal_busio_uart_deinit(busio_uart_obj_t *self) {
 
     gc_free(self->ringbuf);
 
-//    reset_pin_number(self->rx_pin);
-//    reset_pin_number(self->tx_pin);
+//    reset_pin_number(self->rx);
+//    reset_pin_number(self->tx);
```

Comment:
Shouldn't these be uncommented?

Reply:
I was going to do this in a follow up PR, since I wanted to keep this and the reset pins PR separate. I2C, SPI, DigitalIO and PWM all need them too.